modules/ROOT/pages/_partials/include-nav-connected-apps.adoc

```diff
@@ -1 +1,3 @@
-. In the Access Management navigation menu, click *Connected Apps*.
+//Navigation to the Connected Apps page
+
+. In the Access Management navigation menu, click *Connected Apps*.
```

Comment:
Could maybe say "Select Access Management > Connected Apps".

Reply:
They're already in Access Management, so there's no Access Management to select.